google-cloud-examples/src/main/java/com/google/cloud/examples/bigtable/InstanceAdminExample.java

```diff
@@ -0,0 +1,217 @@
+import com.google.cloud.bigtable.admin.v2.models.Instance;
+import com.google.cloud.bigtable.admin.v2.models.Instance.Type;
+import com.google.cloud.bigtable.admin.v2.models.PartialListInstancesException;
+import com.google.cloud.bigtable.admin.v2.models.StorageType;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * An example of using Google Cloud Bigtable.
+ *
+ * <p>This example demonstrates the usage of BigtableInstanceAdminClient to create, configure, and
+ * delete Cloud Bigtable Instances and Clusters.
+ *
+ * <ul>
+ *   <li>creates production instance
+ *   <li>lists instances
+ *   <li>gets instance
+ *   <li>lists clusters
+ *   <li>adds cluster
+ *   <li>deletes cluster
+ *   <li>deletes instance
+ * </ul>
+ */
+public class InstanceAdminExample {
+
+  private static final String CLUSTER = "cluster";
+  private final String clusterId;
+  private final String instanceId;
+  private final BigtableInstanceAdminClient adminClient;
+
+  public static void main(String[] args) throws IOException {
+
+    if (args.length != 1) {
+      System.out.println("Missing required project id");
+      return;
+    }
+    String projectId = args[0];
+
+    InstanceAdminExample instanceAdmin =
+        new InstanceAdminExample(projectId, "ssd-instance", "ssd-cluster");
+    instanceAdmin.run();
+  }
+
+  public InstanceAdminExample(String projectId, String instanceId, String clusterId)
+      throws IOException {
+    this.instanceId = instanceId;
+    this.clusterId = clusterId;
+
+    // [START connecting_to_bigtable]
+    // Creates the settings to configure a bigtable instance admin client.
+    BigtableInstanceAdminSettings instanceAdminSettings =
+        BigtableInstanceAdminSettings.newBuilder().setProjectId(projectId).build();
+
+    // Creates a bigtable instance admin client.
+    adminClient = BigtableInstanceAdminClient.create(instanceAdminSettings);
+    // [END connecting_to_bigtable]
+  }
+
+  public void run() {
+    createProdInstance();
+    listInstances();
+    getInstance();
+    listClusters();
+    addCluster();
+    deleteCluster();
+    deleteInstance();
+    adminClient.close();
+  }
+
+  /** Demonstrates how to create a Production instance within a provided project. */
+  public void createProdInstance() {
+    // Checks if instance exists, creates instance if does not exists.
+    if (!adminClient.exists(instanceId)) {
+      System.out.println("Instance does not exist, creating a PRODUCTION instance");
+      // [START bigtable_create_prod_instance]
+      // Creates a Production Instance with the ID "ssd-instance",
+      // cluster id "ssd-cluster", 3 nodes and location "us-central1-f".
+      CreateInstanceRequest createInstanceRequest =
+          CreateInstanceRequest.of(instanceId)
+              .addCluster(clusterId, "us-central1-f", 3, StorageType.SSD)
+              .setType(Type.PRODUCTION)
+              .addLabel("example", "instance_admin");
```

Comment:
maybe make the label a bit more meaningful:
`.addLabel("department", "accounting")`